### PR TITLE
fix lifecycle link

### LINKED ISSUE #720: Link to R pkgs book for versioning might need to be changed?
@maelle working on versioning text for my work and noticed that the link `https://r-pkgs.org/description.html#version` from the line here https://github.com/ropensci/dev_guide/blob/main/maintenance_releases.Rmd#L16  maybe should be changed to https://r-pkgs.org/lifecycle.html ? `#version` doesn't exist anymore in the original chapter, so I'm guessing you want the lifecycle chapter?

### PR DESCRIPTION
**Pull request by maelle**
Fix #720

### CODE REVIEW

**Comment by mpadge** [comment]
Might it also be worthwhile cross-linking here to [Stats Standard G1.2](https://stats-devguide.ropensci.org/standards.html#documentation)?

File: maintenance_releases.es.Rmd [hunk 1]
```diff
@@ -10,7 +15,7 @@ Ten en cuenta que el proceso de actualización de *NEWS* y de versionado de tu p
 
 ## Versionar {#versioning}
 
-- Recomendamos encarecidamente que los paquetes de rOpenSci utilicen el versionado semántico. Hay una explicación detallada en el [capítulo sobre el archivo *DESCRIPTION* (en inglés)](https://r-pkgs.org/description.html#version).
+- Recomendamos encarecidamente que los paquetes rOpenSci utilicen el versionado semántico. Hay una explicación detallada en el [capitulo "lifecyle" del libro "R packages" (en inglés)](https://r-pkgs.org/lifecycle.html).
```
Comment (yabellini) [inline maintenance_releases.es.Rmd:18]:
```suggestion
- Recomendamos encarecidamente que los paquetes de rOpenSci utilicen el versionado semántico. Hay una explicación detallada en el [capitulo "lifecyle" del libro "R packages" (en inglés)](https://r-pkgs.org/lifecycle.html).
```

**Review by yabellini — COMMENTED** [review]
Looks good.

**Comment by maelle** [comment]
@mpadge to suggest following it?

**Comment by maelle** [comment]
@mpadge I hadn't understood your suggestion, could you please clarify/make an edit suggestion? Thank you!

**Comment by mpadge** [comment]
Unimportant. I thought the PR was about lifecycle, but it's not, so all good :white_check_mark:

**Pull request merged**